contrib/win32/win32compat/win32_dirent.c

```diff
@@ -36,65 +65,68 @@ DIR * opendir(const char *name)
 		return NULL;
 	}
 
-	// add *.* for Windows _findfirst() search pattern
-	swprintf_s(searchstr, MAX_PATH, L"%s\\*.*", wname);
+	/* add *.* for Windows _findfirst() search pattern */
+	if (swprintf(searchstr, MAX_PATH, L"%s\\*.*", wname) == -1) {
+		/* breached MAX_PATH */
```

Comment:
swprintfreturns the number of wide characters stored in buffer, not counting the terminating null wide character. so this comment "breached MAX_PATH" is misleading? 

If we are unable to write to searchstr then we will be inside if loop.